packages/dashboard-hardhat-plugin/yarn.lock

```diff
@@ -0,0 +1,5330 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
This package should not have its own `yarn.lock` (individual packages don't have lockfiles, only the repo root... this doesn't apply to your test HH project directories, though; those can have their own yarn.lock cause they're separate)

Reply:
Addressed in 4b024e8f12d7ed27835e9c76b47f0f41a6c9f388 👍